app/components/AppFooter.vue

```diff
@@ -29,19 +30,19 @@ const unjsLogo = '/unjs.svg'
       <div class="grid md:grid-cols-2 gap-y-6 gap-x-12 md:gap-x-25">
         <div class="grow flex flex-col gap-3 md:gap-6 max-w-sm">
           <NuxtLink to="/" class="flex items-center gap-2 text-xl">
-            <img :src="appConfig.docs.logo" :alt="`${appConfig.docs.name} logo`" class="h-7 w-7" />
+            <img :src="iconLogo" :alt="`${site.name} logo`" class="h-7 w-7" />
             <span class="mt-[2px] font-bold">
-              {{ appConfig.docs.name }}
+              {{ site.name }}
```

Comment:
What is advantage of siteConfig over app config? (if there is no different, I prefer to keep using simple `appConfig` that is tracable to it's source.

Reply:
Thanks for explaining. I wish we could propose this to Nuxt core. It had been discussed long time ago and I really love to see it as a standard practice to use. But coming from an opt-in module (while I fully trust you and love what you made!) cannot be trusted as an standard API we can rely on. (PWA module used to give same experience with sane defaults inferred from env..)

/cc @danielroe